### PR TITLE
1993 perm edit inscri aluno

### PR DESCRIPTION
**Pull request by robertojnior**

### CODE REVIEW

File: aplicacoes/gpa-mae/src/main/webapp/WEB-INF/views/inscricao/auxilio.jsp [hunk 1]
```diff
@@ -56,9 +56,13 @@
 				<form:form id="questionarioForm" role="form"
 					modelAttribute="questionarioAuxilioMoradia"
 					commandName="questionarioAuxilioMoradia"
+<<<<<<< HEAD
```
Comment (regispires) [inline aplicacoes/gpa-mae/src/main/webapp/WEB-INF/views/inscricao/auxilio.jsp]: resolver o conflito

File: aplicacoes/gpa-mae/src/main/webapp/WEB-INF/views/inscricao/iniciacaoAcademica.jsp [hunk 2]
```diff
@@ -45,10 +45,16 @@
 				    onsubmit="return validaHorariosDisponiveisBolsa();"
 					commandName="questionarioIniciacaoAcademica"
 					modelAttribute="questionarioIniciacaoAcademica"
+<<<<<<< HEAD
```
Comment (regispires) [inline aplicacoes/gpa-mae/src/main/webapp/WEB-INF/views/inscricao/iniciacaoAcademica.jsp]: resolver o conflito

File: aplicacoes/gpa-mae/src/main/webapp/WEB-INF/views/selecao/listar.jsp [hunk 3]
```diff
@@ -100,14 +100,31 @@ uri="http://www.springframework.org/security/tags"%>
 								</sec:authorize>	
 
 								<sec:authorize access="hasAnyRole('ROLE_ALUNO')">
+<<<<<<< HEAD
```
Comment (regispires) [inline aplicacoes/gpa-mae/src/main/webapp/WEB-INF/views/selecao/listar.jsp]: resolver o conflito

File: aplicacoes/gpa-mae/src/main/webapp/WEB-INF/views/selecao/listar.jsp [hunk 4]
```diff
@@ -100,14 +100,31 @@ uri="http://www.springframework.org/security/tags"%>
...
+										<c:if test="${selecao.tipoBolsa == inic_acad}">
+										<a id="editar" href="<c:url value="/iniciacaoAcademica/${sessionScope.id}/editar" ></c:url>">
+											<button class="btn btn-info"> Editar <span class="glyphicon glyphicon-pencil"></span></button>
+										</a>
+										<a id="inscrever" href="<c:url value="/iniciacaoAcademica/inscricao" ></c:url>">
+=======
 										<c:if test="${selecao.tipoBolsa == inic_acad && selecao.status == 'INSC_ABERTA'}">
 										<a id="inscrever" href="<c:url value="/iniciacaoAcademica/${selecao.id}/inscricao" ></c:url>">
+>>>>>>> 0a3bf28d1968b7ee10969de23ec49ffe2d5f5f0e
```
Comment (regispires) [inline aplicacoes/gpa-mae/src/main/webapp/WEB-INF/views/selecao/listar.jsp]: resolver o conflito

**Comment by robertojnior** [comment]
Conflitos resolvidos.

**Pull request merged**